src/Evolution/Systems/Cce/WorldtubeDataManager.hpp

```diff
@@ -91,7 +92,8 @@ class MetricWorldtubeDataManager : public WorldtubeDataManager {
           buffer_updater,
       size_t l_max, size_t buffer_depth,
       std::unique_ptr<intrp::SpanInterpolator> interpolator,
-      bool fix_spec_normalization) noexcept;
+      bool fix_spec_normalization,
+      std::optional<double> extraction_radius = std::nullopt) noexcept;
```

Comment:
Is this default necessary? Also, please add a few words about the new argument to the docs.